Server/controllers/articles.controller.js

```diff
@@ -0,0 +1,42 @@
+import Article from "../models/Article.model.js";
+import Response from "../helpers/responses.js";
+
+class Articles {
+  static get = async (req, res) => {
+    Article.find()
+      .then((articles) => {
+        Response.success(res, 200, "Articles fetched successfully", articles);
+      })
+      .catch((err) => Response.error(res, 404, "Articles Not Found"));
```

Comment:
@elvisiraguha This is the server error (server encountered an error and hence couldn't execute the request), 5XX HTTP code is to be used, not 4XX HTTP code. Kindly verify and update this elsewhere.

Reply:
Updated.